CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -1041,10 +1104,12 @@
       </trans-unit>
       <trans-unit id="Portrait" xml:space="preserve">
         <source>Portrait</source>
+        <target>Portret</target>
```

Comment:
Nie wiem, czy tu chodzi dosłownie o portret, czy orientację (geometryczną) pionową

Reply:
@cliss Is this a portrait as in photo of an actor, or is this portrait as in vertical, not landscape?

Reply:
Hm, seems Xcode didn't slurp up my comment for this. Sorry.

It's a photo of a person, and is only "shown" as an accessibility label. So actually it'd be read aloud when voiceover is turned on; it wouldn't generally be seen on-screen.